var/spack/repos/builtin/packages/py-celery/package.py

```diff
@@ -53,24 +54,34 @@ class PyCelery(PythonPackage):
     depends_on("python@2.7:2.8,3.4:", type=("build", "run"))
 
     depends_on("py-setuptools", type=("build", "run"))
-    depends_on("py-setuptools@59.1.1:59.6", type=("build", "run"), when="@5.2.3:")
+    depends_on("py-setuptools@59.1.1:59.6", type=("build", "run"), when="@5.2.3:5.2.4")
 
     depends_on("py-redis@3.2.0:", when="+redis", type=("build", "run"))
     depends_on("py-redis@3.4.1:3,4.0.2:", when="@5.2.3:+redis", type=("build", "run"))
     depends_on("py-sqlalchemy", when="+sqlalchemy", type=("build", "run"))
 
     depends_on("py-click@7.0:7", when="@5.0.0:5.0", type=("build", "run"))
-    depends_on("py-click@8.0.3:8", when="@5.2.0:", type=("build", "run"))
+    depends_on("py-click@8.0.3:8", when="@5.2.0:5.2.99", type=("build", "run"))
```

Comment:
You can specify `when="@5.2"` here since it means any `5.2.x`.  (See docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges.)